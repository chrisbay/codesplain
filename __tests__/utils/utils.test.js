const { makeNode } = require('../../src/utils/utils.js');

describe(`test-utils.js`, () => {
  describe(`makeNode`, () => {
    it(`returns the expected object`, () => {
      const node = makeNode('Some Type', 0, 5, [], []);
      expect(node).toMatchSnapshot();
    });
  });
});
