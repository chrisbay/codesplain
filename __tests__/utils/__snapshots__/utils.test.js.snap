// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test-utils.js makeNode returns the expected object 1`] = `
Object {
  "ast_type": "Some Type",
  "begin": 0,
  "children": Array [],
  "detail": Array [],
  "end": 5,
  "tags": Array [
    "Some Type",
  ],
}
`;
